custom/icds_reports/ucr/data_sources/household_cases.json

```diff
@@ -84,44 +85,38 @@
         "display_name": null,
         "datatype": "string",
         "expression": {
-          "location_id_expression": {
-            "location_id_expression": {
-              "type": "root_doc",
-              "expression": {
-                "datatype": null,
-                "type": "property_name",
-                "property_name": "owner_id"
-              }
+          "location_id": {
+            "expression": {
+              "datatype": null,
+              "type": "property_name",
+              "property_name": "owner_id"
             },
-            "type": "location_parent_id"
+            "type": "root_doc"
           },
-          "type": "location_parent_id"
+          "location_type": "block",
+          "type": "ancestor_location"
         },
         "transform": {},
         "is_nullable": true,
         "is_primary_key": false,
         "column_id": "block_id",
+        "create_index": false,
```

Comment:
why was this added. The default is `false`

Reply:
I think it was a mistake, will remove it.